src/main/java/ch/hearc/todont/models/ToDont.java

```diff
@@ -98,6 +95,25 @@
 
     // METHODS
 
+    /**
+     * Set the closed date to now
+     */
+    public void close() {
+        setDateClosed(Timestamp.from(Instant.now()));
+    }
+
+    /**
+     * Whether the ToDont is already closed
+     */
+    public boolean isClosed() {
+        if (dateClosed == null) {
+            return false;
+        }
+        return Timestamp.from(Instant.now()).after(dateClosed);
```

Comment:
Same comment as above, does it makes sense to be able to close ToDont in the future? Or do we assume that if it has a closed date, it's closed (in case server clock is wrong?)

Reply:
Same answer as above.